app/libs/Model/Registration/Registration.php

```diff
@@ -22,35 +23,54 @@
  * @property string $presentation
  * @property string $note
  * @property DateTimeImmutable|null $deletedAt {default null}
+ * 
+ * @property string $liame {virtual}
+ * @property bool $finvoice {virtual}
+ * @property bool $fvegetarian {virtual}
  */
-class Registration extends Entity
+class Registration extends Entity implements IMessageData
```

Comment:
IMessageData se chci uplne zbavit viz dalsi Issue

Reply:
Počítám, že je důvod to, že kdyby jsi potřeboval více věcí předat do mailu, tak by si stejně musel přepravku vytvořit. Pro tento úkol je to ale v pohodě. Toto bude řešit až ta další Issue.